<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMPSettingsMapAndGame</class>
 <widget class="QWidget" name="PageMPSettingsMapAndGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldIndestructibleVehicles">
     <property name="text">
      <string>Indestructible Vehicles</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelPowerupDurationBlue">
     <property name="text">
      <string>Blue Powerup Duration</string>
     </property>
     <property name="buddy">
      <cstring>fieldPowerupDurationBlue</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelVehicleSet">
     <property name="text">
      <string>Vehicle Set</string>
     </property>
     <property name="buddy">
      <cstring>fieldVehicleSet</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelPowerupDurationRed">
     <property name="text">
      <string>Red Powerup Duration</string>
     </property>
     <property name="buddy">
      <cstring>fieldPowerupDurationRed</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelWeaponSet">
     <property name="text">
      <string>Weapon Set</string>
     </property>
     <property name="buddy">
      <cstring>fieldWeaponSet</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="fieldWeaponSet">
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Map Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Human</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Covenant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">No Snipers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Rocket Launchers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">No Power Weapons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Juggernaut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Slayer Pro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Rifles Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Mid-Range Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Long-Range Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Sniper Rifles</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Melee</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Energy Swords</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Gravity Hammers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">Mass Destruction</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Weapon Set Value">No Weapons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Random</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldTurrets">
     <property name="text">
      <string>Turrets on Map</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldPowerups">
     <property name="text">
      <string>Powerups on Map</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldGrenades">
     <property name="text">
      <string>Grenades on Map</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldAbilities">
     <property name="text">
      <string>Armor Abilities on Map</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="fieldShortcuts">
     <property name="text">
      <string>Shortcuts on Map</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="labelPowerupDurationYellow">
     <property name="text">
      <string>Custom Powerup Duration</string>
     </property>
     <property name="buddy">
      <cstring>fieldPowerupDurationYellow</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="fieldVehicleSet">
     <item>
      <property name="text">
       <string>Map Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mongooses</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warthogs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Aircraft</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Only Aircraft</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Tanks</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Only Tanks</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Light Ground</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Only Light Ground</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Covenant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All Covenant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Human</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All Human</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Vehicles</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All Vehicles</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="fieldPowerupDurationRed">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="fieldPowerupDurationBlue">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="fieldPowerupDurationYellow">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fieldGrenades</tabstop>
  <tabstop>fieldShortcuts</tabstop>
  <tabstop>fieldAbilities</tabstop>
  <tabstop>fieldPowerups</tabstop>
  <tabstop>fieldTurrets</tabstop>
  <tabstop>fieldIndestructibleVehicles</tabstop>
  <tabstop>fieldWeaponSet</tabstop>
  <tabstop>fieldVehicleSet</tabstop>
  <tabstop>fieldPowerupDurationRed</tabstop>
  <tabstop>fieldPowerupDurationBlue</tabstop>
  <tabstop>fieldPowerupDurationYellow</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
